docsrc/getting_started/jetpack.rst

```diff
@@ -0,0 +1,117 @@
+.. _Torch_TensorRT_in_JetPack_6.1
+
+Overview
+##################
+
+JetPack 6.1
+---------------------
+Nvida JetPack 6.1 is the latest production release ofJetPack 6.
+With this release it incorporates:
+CUDA 12.6
+TensorRT 10.3
+cuDNN 9.3
+DLFW 24.0
+
+You can find more details for the JetPack 6.1:
+
+    * https://docs.nvidia.com/jetson/jetpack/release-notes/index.html
+    * https://docs.nvidia.com/deeplearning/frameworks/install-pytorch-jetson-platform/index.html
+
+
+Prerequisites
+~~~~~~~~~~~~~~
+
+
+Ensure your jetson devleoper kit has been flashed with the latest JetPack 6.1. You can find more details on how to flash Jetson board via sdk-manager:
+
+    * https://developer.nvidia.com/sdk-manager
+
+
+check the current jetpack version using
+
+.. code-block:: sh
+
+    apt show nvidia-jetpack
+
+Ensure you have installed JetPack Dev components. This step is required if you need to build on jetson board.
+
+You can only install the dev components that you require: ex, tensorrt-dev would be the meta-package for all TRT development or install everthing.
+
+.. code-block:: sh
+    # install all the nvidia-jetpack dev components
+    sudo apt update
+    sudo apt install nvidia-jetpack
+
+Ensure you have cuda 12.6 installed(this should be installed automatically from nvidia-jetpack)
+
+.. code-block:: sh
+
+    # check the cuda version
+    nvcc --version
+    # if not installed or the version is not 12.6, install via the below cmd:
+    sudo apt update
+    sudo apt install cuda-toolkit-12-6
+
+Ensure libcusparseLt.so exists at /usr/local/cuda/lib64/:
+
+.. code-block:: sh
+
+    # if not exist, download and copy to the directory
+    wget https://developer.download.nvidia.com/compute/cusparselt/redist/libcusparse_lt/linux-sbsa/libcusparse_lt-linux-sbsa-0.5.2.1-archive.tar.xz
```

Comment:
Do we know if this library is validated on Jetson?

Reply:
this is required for pytorch, otherwise got hte following error:
```
>>> import torch
Traceback (most recent call last):
 File “<stdin>“, line 1, in <module>
 File “/home/ubuntu/.local/lib/python3.10/site-packages/torch/__init__.py”, line 289, in <module>
   from torch._C import * # noqa: F403
ImportError: libcusparseLt.so.0: cannot open shared object file: No such file or directory
 
ubuntu@ubuntu:~/.local/lib/python3.10/site-packages/torch$ ldd lib/libtorch_cuda.so | grep libcusparse
libcusparse.so.12 => /usr/local/cuda-12.2/lib64/libcusparse.so.12 (0x0000ffff7c300000)
libcusparseLt.so.0 => not found
```

I searched in the slack channel, DLFW torch build requires download this.
After adding this, at least we can run gpt model in jetson.

Reply:
Hmm its it mentioned in the DLFW PyTorch docs to install this? Would be interesting that this library is required and that isnt mentioned.